extensions/wikia/Email/Controller/CategoryAddController.class.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Email\Controller;
+
+use \Email\EmailController;
+use Email\Check;
+
+class CategoryAddController extends EmailController {
```

Comment:
It is. The Category Add emails use the "UserMailer" category which is the default inherited from `EmailController`.